okio/jvm/src/main/java/okio/AsyncTimeout.kt

```diff
@@ -168,6 +168,7 @@ open class AsyncTimeout : Timeout() {
 
       override fun close() {
         var throwOnTimeout = false
+        enter()
```

Comment:
How long was this missing?!

Reply:
Git history suggests forever?
https://github.com/square/okio/commits/okio_1x/okio/src/main/java/okio/AsyncTimeout.java
https://github.com/square/okio/commit/5880e8947d65b5bdf81ea6f56aff7f5e766c5ed7#diff-cb1656dc5e360018922fbd4b9a342b70R204

Maybe worth a patch back to `okio_1x`?

Reply:
Sure, though no release is currently planned.